x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/watcher/transform/chain/ExecutableChainTransform.java

```diff
@@ -47,7 +45,7 @@ public ChainTransform.Result execute(WatchExecutionContext ctx, Payload payload)
         try {
             return doExecute(ctx, payload, results);
         } catch (Exception e) {
-            logger.error((Supplier<?>) () -> new ParameterizedMessage("failed to execute [{}] transform for [{}]", TYPE, ctx.id()), e);
+            logger.error(() -> org.elasticsearch.core.Strings.format("failed to execute [%s] transform for [%s]", TYPE, ctx.id()), e);
```

Comment:
Nit: might be a bit clearer if we didn't have the full class name here, I don't see another `Strings` class imported.

Reply:
there is `import static org.elasticsearch.common.logging.LoggerMessageFormat.format` which probably deserves a refactoring on its own